.github/workflows/e2e_api_tests.yml

```
@@ -61,14 +61,17 @@ env:
     SHA: ${{ github.event.pull_request.head.sha }}
     SYSTEM_INFO: ./tests/pw/playwright/systemInfo.json
     API_TEST_RESULT: ./tests/pw/playwright-report/api/summary-report/results.json
-    E2E_TEST_RESULT: ./tests/pw/playwright-report/e2e/summary-report/results.json
+    E2E_TEST_RESULT: ./tests/pw/playwright-report/e2e/${{ matrix.group }}/summary-report/results.json
```

actionlint (1.7.4)

context "matrix" is not allowed here. available contexts are "github", "inputs", "secrets", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)